7-vae-mnist.ipynb

```diff
@@ -468,9 +475,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "machine-learning-hats-2021",
+   "display_name": "Python 3",
```

Comment:
Please put the kernel back to "machine-learning-hats-2021".

Reply:
Sorry about that - fixed in the latest commit.